docs/index.asciidoc

```diff
@@ -78,6 +120,19 @@ If you don't have one of these, you can create one by
 registering a new application with Twitter:
 <https://dev.twitter.com/apps/new>
 
+[id="plugins-{type}s-{plugin}-ecs_compatibility"]
+===== `ecs_compatibility`
+
+  * Value type is <<string,string>>
+  * Supported values are:
+    ** `disabled`: does not use ECS-compatible field names (fields might be set at the root of the event)
+    ** `v1`, `v8`: avoids field names that might conflict with Elastic Common Schema (for example, JMS specific properties)
```

Comment:
```suggestion
    ** `v1`, `v8`: avoids field names that might conflict with Elastic Common Schema (for example, Twitter specific properties)
```

Reply:
Good catch!

Reply:
thanks, copy-pasta -> there aren't any Twitter specific properties atm but there might be some day :wink: